### PR TITLE
pyro SuperspreadingSEIRModel process

### PR DESCRIPTION
**Pull request by phoenixAja**
initial exploration using pyro models

### CODE REVIEW

File: exploration/update_pyro_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,1121 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 164,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "region = \"california\"\n",
+    "tree_json = f\"/home/phoenix/code/nf-epigen/jsons/ncov_{region}.json\"\n",
+    "phylogeny = json_to_tree(tree)"
```
Comment (lucymli) [inline exploration/update_pyro_data.ipynb:63]: I had to read in the json first for this to work.
```
    with open(tree_json) as f:
        tree = json.load(f)
     "phylogeny = json_to_tree(tree)"
phylogeny = json_to_tree(tree)
```

**Review by lucymli — COMMENTED** [review]
This is awesome! I want to make some changes to the SuperspreadingSEIR model as that model is a bit too simplistic, but I'll get Fritz's help with that. I'll see if I can schedule a group coding session with his team.

File: exploration/update_pyro_data.ipynb (hunk 1 above)
Comment (phoenixAja) [reply exploration/update_pyro_data.ipynb:63]: oh interesting. I must have had that json object in memory or something like that. feel free to add your new notebook to this PR!

**Comment by phoenixAja** [comment]
yay!!! glad you think it looks good!! I would 100% be interested in a joint coding session with fritz/pyro!! Keep me updated :)

**Comment by phoenixAja** [comment]
do you think it would be ok to add a nextflow step for the pyro stuff with the simpler model? We can update the model later after we meet with pyro team? just so we can get the v0 version out?

**Comment by lucymli** [comment]
That sounds good. Pyro makes it easy to swap out the model, so we can definitely update it, or better, add an option to select from different models.

**Comment by phoenixAja** [comment]
> That sounds good. Pyro makes it easy to swap out the model, so we can definitely update it, or better, add an option to select from different models.

oh yes!! that would be awesome, and would be very cool.

**Comment by phoenixAja** [comment]
oh wait i just realized that we never used the impute infection dates data for the pyro code.. were we supposed to swap out the cleaned timeseries with the imputed infection dates txt file?

**Comment by lucymli** [comment]
ah yes! we do need the cleaned files

**Pull request merged**